CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+# dbt_netsuite v1.1.0
+
+## Schema/Data Change
+**7 total changes • 2 possible breaking changes**
+
+| Data Model(s) | Change type | Old | New | Notes |
+| ------------- | ----------- | ----| --- | ----- |
+| `netsuite2__entity_subsidiary_relationships` | New model | | | Unified view combining customer and vendor subsidiary relationships |
+| `netsuite2__transaction_details` | New columns | | `nexus_id`<br>`nexus_country`<br>`nexus_state`<br>`tax_agency_id`<br>`tax_agency_alt_name`<br>`is_nexus_override`<br>`is_tax_details_override`<br>`tax_point_date`<br>`is_tax_point_date_override` | Nexus-related tax information when var `netsuite2__using_nexuses` is enabled |
```

Comment:
Do we need the note about `netsuite2__using_nexuses`? Especially since the variable doesn't affect things outside of ERTC.

It might be more appropriate to move/copy the breaking change note about `transactions_pass_through_columns` here

Reply:
Sorry, that is old anyway. Nexus doesn't have a var. Added the pass_through_cols bit.